web-quiz-app/src/services/quiz.service.js

```diff
@@ -38,7 +38,7 @@ class QuizDataService {
         return http.get("/instructors/" + instructor_username);
     }
     crossCheckQuizID(quiz_name, instructor_username) {
-        return http.get("/quizzes/" + quiz_name + "/" + instructor_username)
+        return http.get("/quizzes/crossCheck/" + quiz_name + "/" + instructor_username)

```

Comment:
Is crossCheck needed?

Reply:
Prupose of crosscheck was that there can be duplicate quiz names between different instructors (ex. instructor 1 has a quiz called quiz1 and instructor 2 has a quiz also called quiz2). Since Im looking for the quiz based off of the quiz name (what is being stored in localstorage), then I need to confirm that I have the correct quiz ID by checking the current instructor and the quiz they currently selected.

Reply:
Pardon, what I meant (if I understood your response correctly) was that the url path doesn't seem to need the "/crosscheck" part.